src/navigationDevices/rosNavigator/rosNavigator.h

```diff
@@ -69,7 +69,8 @@ class rosNavigator : public yarp::dev::DeviceDriver,
     double                            m_stats_time_last;
 
     bool                              m_moveBase_isAction; // False by default
-    
+    bool                              m_isRecovering; // False by default
```

Comment:
This is not initialized because m_rosSubscriber_recoveryStatus.read(false) could fail

Reply:
```suggestion
    bool                              m_isRecovering = false;
```